eth/beacon/types/states.py

```diff
@@ -158,6 +158,10 @@ def num_validators(self) -> int:
     def num_crosslinks(self) -> int:
         return len(self.latest_crosslinks)
 
+    @property
+    def validators(self) -> Sequence[ValidatorRecord]:
+        return self.validator_registry
```

Comment:
I don't think we need this alias. It might be confusing for the readers about which is the RLP/SSZ field and which is settable.

Reply:
fair enough